Documentation (dpf.docs): ANSYS_ROOT_FOLDER is not an environment variable (#791)
pyansys/pydpf-core: docs/source/user_guide/xmlfiles.rst

on Windows, the path for the root folder for Ansys 2022 R2 likely ends with
``\ANSYS Inc\v222``.

> The ``ANSYS_ROOT_FOLDER`` element defines a variable like an environment variable 

Reviewer: ```suggestion
The ``ANSYS_ROOT_FOLDER`` element defines a variable (similar to an environment variable) 
```

that can be used in the other XML files. For example, you might use it to find required
third-party software.

If the ``ANSYS_ROOT_FOLDER`` element is not defined in the ``DataProcessing.xml``
file, an attempt is made to
find the root folder relative to the ``DataProcessingCore`` DLL or SO file. This
works only if the ``DataProcessingCore.xml`` file is located in its default
location.

If the ``ANSYS_ROOT_FOLDER`` element is still not defined, the root folder is 
determined by reading the ``AWP_ROOT###`` environment variable specific to your 
installed Ansys version. For example, if you are using Ansys 2022 R2, it looks 

> for ``AWP_ROOT222`` to find the root folder.

Reviewer: ```suggestion
for the ``AWP_ROOT222`` environment variable to find the root folder.
```

> ANSYS_ROOT_FOLDER is not an environment variable and can not be set accordingly.

Reviewer 2: ```suggestion
ANSYS_ROOT_FOLDER is not an environment variable and cannot be set accordingly.
```
